pytimize/programs/_linear_program.py

```diff
@@ -718,13 +759,14 @@ def simplex_iteration(self, basis: List[int], show_steps: bool=True, in_place: b
         self.__to_canonical_form(basis, show_steps)
 
         x = self.__compute_basic_solution(basis)
-
         N = [i for i in range(self._A.shape[1]) if i not in basis]
+        k = None
```

Comment:
What do these variables mean? What's going on in this section of code? Stuff like this should be commented, in addition to the method descriptions already added.